client/a8c-for-agencies/hooks/use-track-provisioning-sites.ts

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect, useCallback } from 'react';
+
+const TTL_DURATION = 300000; // 5 minutes in miliseconds
```

Comment:
What do you think about renaming this to `PROVISIONING_TIMEOUT` as it makes it less cryptic and easier to understand?

Reply:
Good suggestion. It has now been applied 👍